include/usearch/usearch.hpp

```diff
@@ -7,26 +7,39 @@
  *  @copyright Copyright (c) 2023
  */
 #pragma once
-#include <algorithm> // `std::sort_heap`
-#include <atomic>    // `std::atomic`
-#include <bitset>    // `std::bitset`
-#include <climits>   // `CHAR_BIT`
-#include <cmath>     // `std::sqrt`
-#include <cstring>   // `std::memset`
-#include <mutex>     // `std::unique_lock` - replacement candidate
-#include <random>    // `std::default_random_engine` - replacement candidate
-#include <unistd.h>  // `open`, `close`
-#include <utility>   // `std::exchange`
-#include <vector>    // `std::vector`
 
 #if defined(WIN32) || defined(_WIN32) || defined(__WIN32__) || defined(__NT__)
+#define NOMINMAX // define this macro to prevent the definition of min/max macros in Windows.h
+#define _USE_MATH_DEFINES
+
+#include <Windows.h>
+
+#define usearch_pack_m
+#define usearch_align_m __declspec(align(64))
+#define WINDOWS
+
 #else
 #include <fcntl.h>    // `fallocate`
 #include <stdlib.h>   // `posix_memalign`
 #include <sys/mman.h> // `mmap`
-#include <sys/stat.h> // `fstat` for file size
```

Comment:
Don't we need `<sys/stat.h>` for `fstat`?

Reply:
It's not removed